core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/TestSolrFilterDelegate.java

```diff
@@ -152,6 +152,18 @@ public void testPropertyIsLike_AnyText_CaseSensitive() {
     END OMIT per DDF-314*/
 
     @Test
+    public void testPropertyIsLikeWildcard() {
+        String searchPhrase = "abc-123*";
+        String expectedQuery = SolrFilterDelegate.WHITESPACE_TOKENIZED_METADATA_FIELD + ":(abc\\-123*)";
+        boolean isCaseSensitive = false;
+
+        SolrQuery isLikeQuery = toTest.propertyIsLike(Metacard.ANY_TEXT, searchPhrase, isCaseSensitive);
+
+        assertThat(isLikeQuery.getQuery(), is(expectedQuery));
+
+    }
```

Comment:
Can we add more to this test? maybe searchPhrase "abc*" or "*123*" to make sure it's tokenizing text without hyphens correctly as well?